### ISSUE TITLE
Illegal Invocation webWorkerRefsByReId

### ISSUE THREAD

**Issue by graysonhicks** [BuilderIO/partytown#39]
Working on a Gatsby + Partytown example w/ GTM and seeing this error:

`TypeError: Illegal invocation.`

I don't see any CORS errors which is the only other instances of this I have found.  The site is up at https://gatsbypartytownfeaturepartytow.gatsbyjs.io/ in debug mode.  The code is here: https://github.com/graysonhicks/gatsby-partytown/tree/feature/partytown.  The error comes from `(anonymous) @ partytown-ww-sw.js:1363` (in debug mode), which is the `catch` here:

```
if (webWorkerRefsByRefId[$refId$]) {
         try {
             const thisArg = deserializeFromMain($instanceId$, [], $thisArg$);
             const args = deserializeFromMain($instanceId$, [], $args$);
                 webWorkerRefsByRefId[$refId$].apply(thisArg, args);
             } catch (e) {
                  console.error(e);
             }
}
```

The GTM setup is the bare minimum and loads the FB pixel script, which is also the bare minimum pixel setup.

**Comment by graysonhicks** [comment]
My next step on this is to try removing the FB pixel to see if it loads without it.  Thinking it could be something in the pixel script.

**Comment by adamdbradley** [comment]
Sorry I haven't been able to dig into this yet. In the meantime we did add more docs that may help out with Facebook Pixel: https://github.com/BuilderIO/partytown/wiki/Forwarding-Events-and-Triggers

**Comment by graysonhicks** [comment]
No rush @adamdbradley .  Hm, then I wonder if the issue is because I am loading the FB pixel _with_ GTM.  I can try adding it with its own integration.

**Comment by adamdbradley** [comment]
Some good fixes went into 0.0.31 that I'm hopefully addressed these issues. Namely the `TrustedHTML` fix that may have been the `TypeError: Illegal invocation.` we found. Would you be able to test out 0.0.31? Thanks

**Comment by graysonhicks** [comment]
Awesome, that got rid of the error!  I'm still not sure anything is actually being called (no GTM events showing or FB pixel events), but I will keep debugging and open separate issues if needed.  Thanks!
